utils/index.ts

```diff
@@ -1,3 +1,5 @@
+import getClassNames from '@/utils/general/getClassNames'
```

Comment:
should be done with a single line 

```suggestion
export { default as getClassNames } from '@/helpers/general/getClassNames'
```